db/import_column_family_job.h

```diff
@@ -75,6 +86,7 @@ class ImportColumnFamilyJob {
   autovector<IngestedFileInfo> files_to_import_;
   VersionEdit edit_;
   const ImportColumnFamilyOptions& import_options_;
+  const std::vector<std::vector<LiveFileMetaData>> metadatas_;
```

Comment:
We can also pass in `std::vector<std::vector<LiveFileMetaData>*>` to avoid copying LiveFileMetaData.